### PR TITLE
Dop 1637: Catch S3 Publish errors

### PR DESCRIPTION
**Pull request by madelinezec**
relevant ticket: https://jira.mongodb.org/browse/DOP-1637

<details here to be completed after my meetings :) >

### CODE REVIEW

File: worker/jobTypes/S3Publish.js [hunk 1]
```diff
@@ -80,9 +81,15 @@ class S3PublishClass {
     try {
       const exec = workerUtils.getExecPromise();
       const command = deployCommands.join(' && ');
-      const { stdout } = await exec(command);
+      const {stdout, stderr } = await exec(command);
```
Comment (skerschb) [inline worker/jobTypes/S3Publish.js:84]: Did we run this through a linter?

**Comment by skerschb** [comment]
This looks ready for integration testing -- I'd just make sure you run it through a linter first.

LGTM

**Review by skerschb — COMMENTED** [review]
Lint and issue the PR against stage and we're good.

**Comment by madelinezec** [comment]
@skerschb so I linted using the simple config, `module.exports = { "extends": "standard" };`

This highlighted many linter issues - so I am concerned that i maybe don't have the right settings / same settings as you? But anyways, take a look

File: worker/jobTypes/S3Publish.js [hunk 2]
```diff
@@ -1,33 +1,30 @@
 const fs = require('fs-extra');
-const workerUtils = require('../utils/utils');
+const workerUtils = require('../utils/utils')
```
Comment (skerschb) [inline worker/jobTypes/S3Publish.js:2]: Did the linter suggest removing the ";"?

**Review by skerschb — APPROVED** [review]
LGTM -- you are using a different linting style than I use...but since I am departing I think we should just use yours anyway!

File: worker/jobTypes/S3Publish.js (hunk 2 above)
Comment (madelinezec) [reply worker/jobTypes/S3Publish.js:2]: yeah the linter wanted me to remove all semicolons - but i didnt do it for all of them, just this one. i can add it back and then merge to stage.

**Pull request merged**